### DISCUSSION TITLE
Unable to see elevation / altitude chart

### DISCUSSION THREAD

**Discussion by JeanGoulet** [mytourbook/mytourbook#1257] [General]
Trying to get the tour chart to show elevation. I can easily turn on/off the chart for speed, heartrate, etc., but elevation never shows up.

Tried a bunch of reset to default, reading documention, but now I have wasted too much time on figuring this out.

Pretty sure this was working before.

No idea why it is greyed out:

![image](https://github.com/mytourbook/mytourbook/assets/16871384/05429e33-4eeb-4331-83d4-d94ef4674c47)

Perhaps somebody has an idea of what could cause this?

Thank you.

**Comment by FJBDev** [comment]
> Trying to get the tour chart to show elevation. I can easily turn off speed, heartrate, etc., but elevation never shows up.
> 

We need more information. Do you have this issue for specific tours ? All tours? 
Can you share a tour for which the issue appears? Do you see any errors in the log ? Which MT version are you using?

> 
> Pretty sure this was working before.
> 

Can you give details on what you mean by "before" ? Before today ? Before an upgrade? If yes, which one?

> Perhaps somebody has an idea of what could cause this?

My idea would be that this imported tour did not have elevation data.

**Comment by JeanGoulet** [comment]
Oh thanks for your insight. Your idea of the imported tour did not have elevation data, is actually correct. I did not think it was possible, because I look at the altimeter frequently on the Garmin Fenix 6, which I got fairly recently, so it's not like it has a broken sensor. However the activity recording that I have been making, was always from the custom activity I made for windsurfing. And that mode does not have elevation displayed on the watch. I was pretty sure it did not matter during recording and saving of an activity, with me expecting the watch to always record every sensor on it regardless of what is displayed or not on the watch. That was based on my previous watch, the Garmin Forerunner 910XT. But it seems Garmin decided to skip over the sensor data that is not displayed on the Fenix 6.

I switched over to Hike mode, and MyTourBook successfully shows the elevation. 

Yet another demonstration of: Assuming nothing, and test everything, to be sure it does the right thing.

**Reply by FJBDev** [reply]
>Oh thanks for your insight. Your idea of the imported tour did not have elevation data, is actually correct. 

Good to know. I'm glad to read that switching to Hiking mode resloved the issue.
